core/meta/src/TCling.cxx

```diff
@@ -4980,6 +4996,14 @@ static cling::Interpreter::CompilationResult ExecAutoParse(const char *what,
       "#undef ClassDef\n"
       "#define ClassDef(name,id) \\\n"
       "_ClassDef_(name,id,virtual,) \\\n"
+      "static int DeclFileLine() { return __LINE__; }\n"
+      "#undef ClassDefNV\n"
+      "#define ClassDefNV(name, id)\\\n"
+      "_ClassDef_(name,id,,)\\\n"
+      "static int DeclFileLine() { return __LINE__; }\n"
+      "#undef ClassDefOverride\n"
+      "#define ClassDefOverride(name, id)\\\n"
+      "_ClassDef_(name,id,,override)\\\n"
       "static int DeclFileLine() { return __LINE__; }\n";
    if (!header) {
       // This is the complete header file content and not the
```

Comment:
Could you move these two string literals (if they are identical) into say a common gNonInterpreterClassDef? Like that we don't duplicate them; a bug in them only needs to be fixed once.

Thanks!